extras/otel/otel-collector-config.yaml

```diff
@@ -68,17 +74,28 @@ exporters:
     endpoint: "api.honeycomb.io:443"
     headers:
       "x-honeycomb-team": ${env:HONEYCOMB_API_KEY}
+  awss3/metrics:
```

Comment:
How is the access to this bucket handled ?

Reply:
On AWS it will just use the instance metadata endpoint. The [otel-s3-doc] says it uses the default from the [go-sdk] which fallbacks to the "EC2 Instance Role Credentials" [1]

[otel-s3-doc]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/awss3exporter#aws-credential-configuration
[go-sdk]: https://docs.aws.amazon.com/sdk-for-go/api/#hdr-Configuring_Credentials
[1]: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html

Reply:
If it fallbacks to the EC2 role, then does it mean that the user application containers will inherit these permissions as well without having to do anything ?

Reply:
Oh dang, yes!! I recommend that we then rather find a way to install the collector directly on the VM :smile: